backend/backend.go

```diff
@@ -247,11 +251,11 @@ func (backend *Backend) CreateAndAddAccount(
 	switch specificCoin := coin.(type) {
 	case *btc.Coin:
 		account = btc.NewAccount(specificCoin, backend.arguments.CacheDirectoryPath(), code, name,
```

Comment:
account does not need socksProxy in general.

Better to pass it to `*\.NewCoin()`, as the coin instances perform the communication.

This will reduce the diff quite a bit.

Reply:
I can easily move it there, but don't really think this will reduce the diff. Account just holds it in its struct until coin.Initialize() is called. My hunch is this will actually be more code with more repetition, since we call `NewCoin` way more than `NewAccount`.

Reply:
Ok the diff might not be smaller, but Coin is still the better/correct home for it ;) It is confusing to pass it through account just to pass it to coin.Initialize().

Also I slightly dislike argument to Initialize() functions, imho the instance having the pre-init data it needs at creation time is a bit nicer / concentrated.